email-verifier/src/api.ts

```
@@ -100,6 +100,39 @@ export const bulk = async (emails: string[], hostname: string) => {
+      await sendRequest({
+        url: `${hostname}/verifier/webhook`,
+        method: 'POST',
+        body: {
+          emails: invalidEntries.map((email) => ({
+            email,
+            status: EMAIL_VALIDATION_STATUSES.INVALID,
+          })),
+        },
+      });
+    } catch (e) {
+      throw e;
```

Biome

[error] The catch clause that only rethrows the original error is redundant.

These unnecessary catch clauses can be confusing. It is recommended to remove them.

(lint/complexity/noUselessCatch)